pkg/secrets/secrets.go

```diff
@@ -157,12 +158,12 @@ func SecretMountsWithUIDGID(mountLabel, containerWorkingDir, mountFile, mountPre
 	_, err := os.Stat("/etc/system-fips")
 	if err == nil {
 		if err := addFIPSModeSecret(&secretMounts, containerWorkingDir); err != nil {
-			logrus.Warnf("error adding FIPS mode secret to container: %v", err)
+			logrus.Errorf("error adding FIPS mode secret to container: %v", err)
```

Comment:
Will this print on systems without FIPS mode enabled?

Reply:
Seems that it won't. Looks fine, then.

Reply:
Only will print if you have a /etc/system-fips file and something goes wrong with setting up the secret, which is what we want.  You really do not want /etc/system-fips file to exist on your system, since it slows down your encrypted sessions.